templates/base.html.twig

```diff
@@ -94,7 +94,8 @@
             wikiUrl: "{{ wiki_url }}",
             assetsPath: "{{ asset('assets') }}",
             {# This var is used for the Universal Language Selector. #}
-            languages: {{ all_langs()|json_encode|raw }}
+            languages: {{ all_langs()|json_encode|raw }},
+            user: {{ username|json_encode|raw }}
```

Comment:
It'd work the same, but instead of passing `username` in from the controller, there is the `logged_in_user()` Twig function that is provided by the bundle. Could be used here to set a `loggedIn` attribute.